packages/storybook-utils/package.json

```diff
@@ -30,16 +30,20 @@
   "peerDependencies": {
     "@sit-onyx/icons": "workspace:^",
     "@sit-onyx/shared": "workspace:^",
+    "@storybook/addon-actions": "^8.4.5",
```

Comment:
```suggestion
    "@storybook/addon-actions": ">= 8.3.0",
```

Since its a peerDependency, we should use a wider range (`>=`), otherwise projects are forced to use this version

Reply:
done